plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java

```diff
@@ -59,17 +63,23 @@ public void importOrderFile(Object importOrderFile) {
 		addStep(ImportOrderStep.createFromFile(getProject().file(importOrderFile)));
 	}
 
+	/** Use {@link #eclipse()} instead */
```

Comment:
The idea itself is very nice and I will definitely use this approach in future when describing more complex use cases. 
But honestly I do not think that this would fit here. I saw that you guys try avoid superfluous comments where code is self explaining. I think we agree that the usage of `EclipseConfig` is quite self explaining.  
But that's the final comment from my side. The final decision is yours. Let me know.